tests/test_process.py

```diff
@@ -687,6 +716,51 @@ def test_module_name(self) -> None:
         out = self.run_command("python -m coverage")
         assert "Use 'coverage help' for help" in out
 
+    @pytest.mark.skipif(env.WINDOWS, reason="This test is not for Windows")
+    def test_save_signal_usr1(self) -> None:
```

Comment:
I think maybe we don't need this test since `test_save_signal` covers it?  Same for `test_save_signal_kill`, though maybe there's some subtlety I'm overlooking?